chai.js

```diff
@@ -10017,380 +9992,7 @@ function isPrimitive(value) {
   return value === null || typeof value !== 'object';
 }
 
-},{"type-detect":36}],36:[function(require,module,exports){
-'use strict';
```

Comment:
Looks like something funky has happened here. Is it maybe type-detect was deduped?

Reply:
Okay, looks like type-detect was deduped, 2 copies were present prior to this commit.

Reply:
Yikes, what the hell is *that*? :|

Erm, hang on, let me look into this. I actually did a fresh checkout and ran `npm install .` after cloning. Then followed up with the `make release-patch` (as you explained earlier), and there were no errors displayed in my terminal, so...

Reply:
It's all good. `deep-eql` and `chai` both rely on `type-detect` - but up until now they each relied on a different version. This is a good thing. No worries!